railties/lib/rails/generators/rails/app/templates/Dockerfile.tt

```diff
@@ -0,0 +1,24 @@
+# Make sure it matches the Ruby version in .ruby-version and Gemfile
+FROM ruby:<%= Gem.ruby_version %>
+
+# Install libvips for Active Storage preview support
+RUN apt-get update -qq && apt-get install -y build-essential libvips
```

Comment:
:thinking: Would it make sense to skip this when `skip_active_storage` is used (when using `minimal` skeleton for example)?

Reply:
Yes